src/main/java/org/fusesource/jansi/internal/OSInfo.java

```diff
@@ -169,7 +182,7 @@ private static String readFully(InputStream in) throws IOException {
         while ((readLen = in.read(buf, 0, buf.length)) >= 0) {
             b.write(buf, 0, readLen);
         }
-        return b.toString();
+        return b.toString("UTF-8");
```

Comment:
`StandardCharsets.UTF_8` ?

Reply:
> `StandardCharsets.UTF_8` ?

`ByteArrayOutputStream::toString(Charset)`, so we cannot use it here.